src/core/library.h

```diff
@@ -299,6 +304,19 @@ QuicPartitionIndexDecrement(
     }
 }
 
+_IRQL_requires_max_(DISPATCH_LEVEL)
+inline
+void
+QuicPerfCounterAdd(
```

Comment:
Might be worth having a `#define` for a `QuicPerfCounterIncrement` and `QuicPerfCounterDecrement` but I leave it up to you.